### ISSUE TITLE
initialize() was called twice

### ISSUE THREAD

**Issue by jiajunhui** [NordicSemiconductor/Android-BLE-Library#268]
I'm using version 2.2.4
If the device is already bound, initialization will be called multiple times.
I found that when the device is in the bound state, the BleManagerHandler initiates a discovery service again
```java
case BluetoothDevice.BOND_BONDED:
	log(Log.INFO, "Device bonded");
	postCallback(c -> c.onBonded(device));
	postBondingStateChange(o -> o.onBonded(device));
	if (request != null && request.type == Request.Type.CREATE_BOND) {
		request.notifySuccess(device);
		request = null;
		break;
	}
	// If the device started to pair just after the connection was
	// established the services were not discovered.
	if (!servicesDiscovered && !serviceDiscoveryRequested) {
		serviceDiscoveryRequested = true;
		post(() -> {
			log(Log.VERBOSE, "Discovering services...");
			log(Log.DEBUG, "gatt.discoverServices()");
			bluetoothGatt.discoverServices();
		});
		return;
	}
```

**Comment by jiajunhui** [comment]
On Samsung's S10 device, the initialization method is called repeatedly instead of twice

**Comment by philips77** [comment]
Thank you for creating this issue. Could you paste the logs here?

**Comment by philips77** [comment]
Related to #120.

**Comment by philips77** [comment]
This might have been fixed by 65e08d5.

**Comment by philips77** [comment]
Perhaps fixed internally and will be released as 2.3. Closing this issue for now, please reopen if 2.3 still fails.
